src/Sfa.Das.ApprenticeshipInfoService.Api/Controllers/ProvidersV3Controller.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Web.Http;
+using System.Web.Http.Description;
+using Microsoft.Web.Http;
+using Sfa.Das.ApprenticeshipInfoService.Core.Helpers;
+using Sfa.Das.ApprenticeshipInfoService.Core.Models;
+using Sfa.Das.ApprenticeshipInfoService.Core.Services;
+using SFA.DAS.Apprenticeships.Api.Types.V3;
+using SFA.DAS.NLog.Logger;
+using Swashbuckle.Swagger.Annotations;
+
+namespace Sfa.Das.ApprenticeshipInfoService.Api.Controllers.V3
+{
+    [ApiVersion("3.0")]
+    [RoutePrefix("v{version:apiVersion}")]
+    public class ProvidersV3Controller : ApiController
```

Comment:
Should this be two endpoints or does it require having a separate endpoint for both standards and frameworks

Reply:
I'm on the fence. 🤺 
It's currently in keeping with the other endpoints but you're right in that there is no framework/standard specific data coming back in the results.

Reply:
Have made the change to a single endpoint 😃